roles/openldap/defaults/main.yml

```diff
@@ -0,0 +1,52 @@
+keystone:
```

Comment:
keystone_user_name is a user used by keystone to connect to openldap, similar to the user keystone uses to connect to mysql. As such it should be specified in one place and used by both roles, I think we can change the name to ldap_connect_user and specify it in openldap role.

I checked the mysql connection param in keystone.conf, seems it assumes the database/user name is always keystone.
[sql]
connection=mysql://<b>keystone</b>:{{ secrets.db_password }}@{{ endpoints.db }}/<b>keystone</b>?charset=utf8